.circleci/config.yml

```diff
@@ -0,0 +1,145 @@
+version: 2.1
+
+orbs:
+  win: circleci/windows@2.2.0
+
+references:
+  environment: &ENVIRONMENT
+    GOMAXPROCS: 4
+    GO111MODULE: "on"
+    GOPROXY: https://proxy.golang.org/
+    TEST_RESULTS_DIR: &TEST_RESULTS_DIR /tmp/test-results
+    WIN_TEST_RESULTS: &WIN_TEST_RESULTS c:\Users\circleci\AppData\Local\Temp\test-results
+
+commands:
+  run-gotests:
+    parameters:
+      cmd:
+        type: string
+      platform:
+        type: string
+    steps:
+      - run:
+          name: "Run go tests"
+          command: |
+            PACKAGE_NAMES=$(go list ./... | circleci tests split --split-by=timings --timings-type=classname)
+            echo "Running $(echo $PACKAGE_NAMES | wc -w) packages"
+            echo $PACKAGE_NAMES
+            << parameters.cmd >> --format=short-verbose --junitfile $TEST_RESULTS_DIR/hil/gotestsum-report.xml -- -p 2 -cover -coverprofile=<< parameters.platform >>_cov_$CIRCLE_NODE_INDEX.part $PACKAGE_NAMES
+        
+jobs:
+  linux-tests:
+    docker:
+      - image: circleci/golang:<< parameters.go-version >>
```

Comment:
Circle is publishing a new family of Golang images [`cimg/go:<<version>>`](https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/). Probably good to start using them sooner rather than later.

Reply:
ah, good catch! I'll update this